lib/openstax/salesforce/active_force.rb

```diff
@@ -7,10 +7,13 @@ class << self
     # to allow the Client to be successfully instantiated.
     alias_method :original_sfdc_client, :sfdc_client
     def sfdc_client
-      unless original_sfdc_client.is_a?(OpenStax::Salesforce::Client)
+      if original_sfdc_client.is_a?(OpenStax::Salesforce::Client)
         self.sfdc_client = OpenStax::Salesforce::Client.new
+      elsif original_sfdc_client.is_a?(OpenStax::Salesforce::BulkApi)
```

Comment:
The `original_sfdc_client` is an alias of the `sfdc_client` method that comes from ActiveForce, so it should always be an ActiveForce class (unless we apply this monkeypatch twice somehow).

I think we'll instead need to pick the client class to return here based on some configuration variable?